projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -146,6 +146,7 @@
 import org.batfish.grammar.assertion.AssertionCombinedParser;
 import org.batfish.grammar.assertion.AssertionExtractor;
 import org.batfish.grammar.assertion.AssertionParser.AssertionContext;
+import org.batfish.grammar.cisco.CiscoParser.Color_setterContext;
```

Comment:
Looks like you added an unused import -- this will make Ari sad later.

(The Java Findbugs plugin, once fully supported, will be able to catch this in Travis. The issue right now is that there are so many bugs :).